src/components/Accordion.js

```diff
@@ -0,0 +1,75 @@
+import "../polyfills/includes.polyfill";
+
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+document.addEventListener(
+  "click",
+  onDocumentClick => {
+    const { target } = onDocumentClick;
+
+    if (target.className.includes("dg_allitems")) {
+      allMenuItemsAction(target);
+    } else if (target.className.includes("dg_accordian-btn")) {
+      menuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+const menuAction = element => {
+  var mainDiv = element.closest(".dg_accordion");
+  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
+  var childDiv = element.nextElementSibling;
+  var menuID = childDiv.id;
+
+  for (let i = 0; i < menuItems.length; i++) {
+    var menuItem = menuItems[i];
+
+    if (menuID !== menuItems[i].id) {
+      menuItem.className = mainDiv.className.includes("dg_allowmutlipleopen")
+        ? menuItem.className
+        : menuState("close");
+    } else {
+      menuItem.className.includes(menuOpen)
```

Comment:
i could have been more clear (not blocking) but i would have recommended something like.

```js
const isMenuOpen = menuItem.className.includes(menuOpen);

menuItem.className = menuState(isMenuOpen ? "close" : "open");
menuItem.setAttribute("aria-expanded", !isMenuOpen);
menuItem.closest(".dg_menuitem").className = `dg_menuitem ${isMenuOpen ?  "" : "open"}`.trim();
```

```

Reply:
moved this into its own function since this is redundant code.